code/modules/surgery/generic.dm

```diff
@@ -52,6 +52,7 @@
 	"<span class='notice'> You have made an incision on [target]'s [affected.name] with \the [tool].</span>",)
 	affected.open = 1
 	affected.status |= ORGAN_BLEEDING
+	target.bad_external_organs += affected
```

Comment:
has to be done, other wise the organ code doesn't run: https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/mob/living/carbon/human/human_organs.dm#L35

even though I know it's kinda ugly I don't really see a way around it